Code/GraphMol/testChirality.cpp

```diff
@@ -2818,18 +2818,18 @@ class TestAssignChiralTypesFromMolParity {
         {Atom::CHI_UNSPECIFIED, 0},
         {Atom::CHI_OTHER, 0}};
     MolOps::assignChiralTypesFrom3D(*d_rwMol);
-    for (auto ai = d_rwMol->beginAtoms(); ai != d_rwMol->endAtoms(); ++ai) {
-      int parity = parityMap.at((*ai)->getChiralTag());
-      (*ai)->setProp(common_properties::molParity, parity);
-      (*ai)->setChiralTag(Atom::CHI_UNSPECIFIED);
+    for (const auto atom : d_rwMol->atoms()) {
+      int parity = parityMap.at((atom)->getChiralTag());
+      (atom)->setProp(common_properties::molParity, parity);
+      (atom)->setChiralTag(Atom::CHI_UNSPECIFIED);
     }
   }
   void fillBondDefVect() {
-    for (auto bi = d_rwMol->beginBonds(); bi != d_rwMol->endBonds(); ++bi) {
-      d_bondDefVect.emplace_back(BondDef((*bi)->getBeginAtomIdx(),
-                                         (*bi)->getEndAtomIdx(),
-                                         (*bi)->getBondType()));
-    }
+	for (const auto bond : d_rwMol->bonds()) {
```

Comment:
I forgot to mention this when we talked: we use clang-format to make sure that the code is consistently formatted. You should too.

In order to make your life easier (and reduce the chances that you forget to do this), I would suggest to either set your editor up to run clang-format when you save or add a pre-commit hook for git to run it whenever you do a commit (I think think this is possible).

There's a configuration file in $RDBASE that clang-format should pick up automatically so that the correct formatting rules are applied.

Reply:
Another general tip I forgot to mention: the RDKit CI builds run with compiler warnings treated as errors (in an effort to keep the code clean). It's probably a good idea for you to do this too. 

If you're using g++, the compiler flags to use are: `-Wall -Werror`. Ask (via email is probably best) if you have questions about setting that up.